js/pages/cohort-definitions/cohort-definition-manager.js

```diff
@@ -465,6 +465,9 @@ define(['jquery', 'knockout', 'text!./cohort-definition-manager.html',
 							if (source) {
 								// only bother updating those sources that we know are running
 									if (this.isSourceRunning(source)) {
+										if(source.status() !== info.status) {
+											jobDetailsService.setJobListMutated();
```

Comment:
Cannot we avoid imperative dirty declaration in favor of using pure observables?